### PR TITLE
feat(Organizations): add organizations policy resource

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  add organizations policy resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  add organizations policy resource
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
go test ./huaweicloud/services/acceptance/organizations -v -run TestAccPolicy_basic -timeout 360m 
=== RUN   TestAccPolicy_basic
=== PAUSE TestAccPolicy_basic
=== CONT  TestAccPolicy_basic
--- PASS: TestAccPolicy_basic (9.50s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/organizations     10.513s
```

### CODE REVIEW

File: docs/resources/organizations_policy.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  type     = "tag_policy"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, String) Specifies the name to be assigned to the policy. It can contain 1 to 64 characters, only
+  english and chinese letters, digits, underscore (_), hyphens (-) and spaces are allowed and the first and last
```
Comment (Jason-Zhang9309) [inline docs/resources/organizations_policy.md]:
```suggestion
  English and Chinese letters, digits, underscore (_), hyphens (-) and spaces are allowed and the first and last
```

File: huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+				ForceNew:    true,
+				Description: `Specifies the type of the policy to be created.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the description to be assigned to the policy.`,
+			},
+			"tags": {
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go]: use common.TagsSchema

File: huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+			"type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the type of the policy to be created.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go]: remove `Computed:    true`  if the description can be updated to empty

File: huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+		_, err = updatePolicyClient.Request("PATCH", updatePolicyPath, &updatePolicyOpt)
+		if err != nil {
+			return diag.Errorf("error updating Organizations policy: %s", err)
+		}
+	}
+
+	if d.HasChange("tags") {
+		err = updateTags(d, updatePolicyClient, policiesType, d.Id(), "tags")
+		if err != nil {
+			return diag.FromErr(err)
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go]: looks the error message is fixed to organizations tags, please check it

File: docs/resources/organizations_policy.md (hunk 1 above)
Comment (houpeng80) [reply docs/resources/organizations_policy.md]: fixed

File: huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go]: done

File: huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go (hunk 3 above)
Comment (houpeng80) [reply huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go]: done

File: huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go (hunk 4 above)
Comment (houpeng80) [reply huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go]: done

File: docs/resources/organizations_policy.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+Manages an Organizations policy resource within HuaweiCloud.
+
+## Example Usage
+
+### Create service control policy
+
+```hcl
+resource "huaweicloud_organizations_policy" "scp_policy"{
+  name     = "test_policy_name"
+  contenet = "{\"Version\":\"5.0\",\"Statement\":[{\"Effect\":\"Deny\",\"Action\":[]}]}"
```
Comment (ShiChangkuo) [inline docs/resources/organizations_policy.md]: it's better to use jsonencode function or heredoc
https://jeffbrown.tech/terraform-heredoc-jsonencode/

**Comment by houpeng80** [comment]
go test ./huaweicloud/services/acceptance/organizations -v -run TestAccPolicy_basic -timeout 360m
=== RUN   TestAccPolicy_basic
=== PAUSE TestAccPolicy_basic
=== CONT  TestAccPolicy_basic
--- PASS: TestAccPolicy_basic (9.95s)
PASS                                 
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/organizations     11.016s

File: docs/resources/organizations_policy.md (hunk 5 above)
Comment (houpeng80) [reply docs/resources/organizations_policy.md]: done

**Pull request merged**